docs/content/components/labels.md

```diff
@@ -190,22 +190,6 @@ You can also have icons and emoji in counters. Or use utilities for counters wit
 <span class="Counter mr-1 bg-purple text-white">22</span>
 ```
 
-## Small counters
-
-Use the `Counter--small` modifier class to reduce the size of a counter.
-
-```html live
-<span class="Counter mr-1 Counter--gray-light">1</span>
-<span class="Counter mr-1">23</span>
-<span class="Counter mr-1 Counter--gray">456</span>
-<span class="Counter mr-1">1.5K</span>
-<div class="my-2"></div>
-<span class="Counter Counter--small mr-1 Counter--gray-light">1</span>
-<span class="Counter Counter--small mr-1">23</span>
-<span class="Counter Counter--small mr-1 Counter--gray">456</span>
-<span class="Counter Counter--small mr-1">1.5K</span>
-```
-
```

Comment:
I see you deleting the docs here, but I didn't see `Counter--small` deleted in the css. Missed or already deleted before?

Reply:
Yep, the `.Counter--small` doesn't exist anymore and I just forgot to remove it from the docs. We might can consider adding a `.Counter--large`.